packages/conversation/tests/rest/v1/webhooks-events/webhooks-events.steps.ts

```diff
@@ -0,0 +1,269 @@
+import { Given, Then, When } from '@cucumber/cucumber';
+import { ConversationCallbackWebhooks, Conversation } from '../../../../src';
+import assert from 'assert';
+import { IncomingHttpHeaders } from 'http';
+
+const APP_SECRET = 'CactusKnight_SurfsWaves';
+let conversationCallbackWebhook: ConversationCallbackWebhooks;
+let rawEvent: any;
+let event: Conversation.ConversationWebhookEvent;
+let formattedHeaders: IncomingHttpHeaders;
+
+const processEvent = async (response: Response) => {
+  formattedHeaders = {};
+  response.headers.forEach((value, name) => {
+    formattedHeaders[name.toLowerCase()] = value;
+  });
+  rawEvent = await response.text();
+  rawEvent = rawEvent.replace(/\s+/g, '');
+  event = conversationCallbackWebhook.parseEvent(JSON.parse(rawEvent));
+};
+
+Given('the Conversation Webhooks handler is available', () => {
+  conversationCallbackWebhook = new ConversationCallbackWebhooks(APP_SECRET);
+});
+
+Then('the Conversation event header contains a valid signature', () => {
+  assert.ok(conversationCallbackWebhook.validateAuthenticationHeader(formattedHeaders, rawEvent));
+});
+
+When('I send a request to trigger a "CAPABILITY" event', async () => {
```

Comment:
These scenarios are currently commented onto mock server PR

Reply:
Fixed, sorry for that : https://github.com/sinch/sinch-sdk-mockserver/pull/19/commits/7ca457e6ca14b55c1a42470f397d7848cc03b34c
(approval needed again 🫤)